backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -96,7 +96,7 @@ def per_page_dropdown
 
         select_tag(:per_page,
           options_for_select(per_page_options, params['per_page'] || per_page_default),
-          { id: "js-per-page-select", class: "form-control pull-right" })
+          { class: "form-control pull-right js-per-page-select" })
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.